src/GameManager.cpp

```diff
@@ -3512,6 +3512,21 @@ bool GameManager::stepstype_is_multiplayer(StepsType st)
 		category == StepsTypeCategory_Routine;
 }
 
+int GameManager::get_num_pads_for_stepstype(StepsType st)
+{
+	switch(GetStepsTypeInfo(st).m_StepsTypeCategory)
+	{
+		case StepsTypeCategory_Single:
+			return 1;
+		case StepsTypeCategory_Double:
+		case StepsTypeCategory_Couple:
+		case StepsTypeCategory_Routine:
+			return 2;
```

Comment:
Personal note: eventually look into allowing Routine to be more than 2 since 3 and 4 person charts (heck, even 5) are possible.